### DISCUSSION TITLE
updating dropwizard to 2.0.13 from 1.3.24 causes `package org.assertj.core.api does not exist`

### DISCUSSION THREAD

**Discussion by aalsanie** [dropwizard/dropwizard#4767] [General]
I'm updating my dropwizard project to latest version. I use `dropwizard-testing` for assertions. for example:
`assertThat(response("/get", Boolean.class)).isNotNull().isEqualTo(true);`
using assertj:
`import static org.assertj.core.api.Assertions.assertThat`
this used to work fine in `1.3.24` but causing compilation error on update.
pom.xml:
```
<dependency>
   <groupId>io.dropwizard</groupId>
   <artifactId>dropwizard-testing</artifactId>
</dependency>
```

**Comment by joschi** [comment]
You have to add AssertJ explicitly if you're using it in your tests, since it's only used internally for tests in `dropwizard-testing` and not exposed whatsoever.

```
<dependency>
  <groupId>org.assertj</groupId>
  <artifactId>assertj-core</artifactId>
  <scope>test</scope>
</dependency>
```

See also #3202 and #3215.
